homeassistant/components/fritzbox/strings.json

```
@@ -20,12 +20,12 @@
     },
     "abort": {
       "already_in_progress": "[%key:common::config_flow::abort::already_in_progress%]",
-      "already_configured": "This AVM FRITZ!Box is already configured.",
-      "not_found": "No supported AVM FRITZ!Box found on the network.",
+      "already_configured_device": "[%key:common::config_flow::abort::already_configured_device%]",
```

The key can stay as it was.
```suggestion
      "already_configured": "[%key:common::config_flow::abort::already_configured_device%]",
```